app/src/main/java/com/example/cocktaildb/screen/auth/AuthFragment.kt

```diff
@@ -0,0 +1,103 @@
+package com.example.cocktaildb.screen.auth
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.example.cocktaildb.MainActivity
+import com.example.cocktaildb.R
+import com.example.cocktaildb.data.repository.AuthRepository
+
+class AuthFragment : Fragment(), AuthContract.View {
+
+    private lateinit var presenter: AuthPresenter
+    private lateinit var etEmail: EditText
+    private lateinit var etPassword: EditText
+    private lateinit var btnSignIn: TextView
+    private lateinit var tvForgotPassword: TextView
+    private lateinit var btnGoogle: ImageView
```

Comment:
mình sử dụng [viewBinding](https://github.com/awesome-academy/HN_Pra_2025_Android_Cocktail/pull/3/files#diff-8cff73265af19c059547b76aca8882cbaa3209291406f52df1dafbbc78e80c46R46) mà em nhỉ dùng trực tiếp chứ không cần khai báo lại.


ref: https://viblo.asia/p/su-dung-view-binding-thay-the-cho-findviewbyid-trong-android-aWj53DwPK6m